### PR TITLE
Modification on My Api Page side bar 

### PR DESCRIPTION
**Pull request by Yagazie-davidson**
## Contributions
- Implementation of the drop downs and the tabs.
- Fixed a bug on the Organization page (I just noticed it)
## TODO

- Analytics page

### CODE REVIEW

**Review by taiwoAkindele — APPROVED** [review]

File: src/App.js [hunk 1]
```diff
@@ -18,83 +18,84 @@ import OrganizationPage from './pages/OrganizationPage'
...
+	const logInUser = () => {
+		if (accessToken) {
+			const accessToken = cookies.get("accessToken");
+			const refreshToken = cookies.get("refreshToken");
+			const profileId = cookies.get("profileId");
+			const fullName = cookies.get("fullName");
+			const email = cookies.get("email");
+			const data = { accessToken, refreshToken, profileId, fullName, email };
+			dispatch(login(data));
+			navigate(`/user/${profileId}`);
```
Comment (PhilemonBrain) [inline src/App.js:46]: why are u navigating??? @Yagazie-davidson

File: src/components/OrgTeammates.jsx [hunk 2]
```diff
@@ -1,68 +1,67 @@
...
+export default function OrganizationTeammates() {
+	const { id } = useParams();
+	const classes = useStyles();
+	const [orgTeam, setOrgTeam] = useState([]);
+	async function getOrganizationUsers() {
+		const res = await fetch(`${base_url}/organisation/users/${id}`);
+		const data = await res.json();
+		setOrgTeam(data.data);
+	}
+	console.log(orgTeam);
```
Comment (PhilemonBrain) [inline src/components/OrgTeammates.jsx:34]: remove these logs when you're pushing

**Review by PhilemonBrain — CHANGES_REQUESTED** [review]

File: src/App.js (hunk 1 above)
Comment (taiwoAkindele) [inline src/App.js:46]: The useNavigate hook is not imported, perhaps it was deleted by mistake.

**Review by taiwoAkindele — APPROVED** [review]

File: src/App.js (hunk 1 above)
Comment (Yagazie-davidson) [reply src/App.js:46]: This isn't my code

Comment (PhilemonBrain) [reply src/App.js:46]: then there is an issue. it means you're not updated on the main branch @Yagazie-davidson

**Review by PhilemonBrain — CHANGES_REQUESTED** [review]

**Pull request closed without merging**